finesse/gui/temp_control.py

```diff
@@ -144,15 +144,27 @@ def _update_figure(self, event) -> None:
 
         self._canvas.draw()
 
+    def _plot_bb_temps(self, values: list[Decimal], time: float) -> None:
```

Comment:
Logically I feel these arguments should be the other way round. It's a bit odd to have your `y` coming after your `x`.